htdocs/modulebuilder/template/core/modules/modMyModule.class.php

```diff
@@ -229,7 +229,7 @@ public function __construct($db)
 			'tabhelp'=>array(array('code'=>$langs->trans('CodeTooltipHelp'), 'field2' => 'field2tooltip'), array('code'=>$langs->trans('CodeTooltipHelp'), 'field2' => 'field2tooltip'), ...),
 		);
 		*/
-
+		/*Begin dictionaries*/
```

Comment:
Can you replace the /*Begin dictionaries*/
with 
/* BEGIN MODULEBUILDER DICTIONARIES */
and
/* END MODULEBUILDER DICTIONARIES */

So it follow same rules than other sections...

Reply:
yes of course